Dockerfile

```diff
@@ -106,18 +106,6 @@ RUN rm -rf ./mathesar/tests ./db/tests
 RUN rm -rf ./docs
 
 
-#=========== STAGE: DEMO =====================================================#
-
-FROM common AS demo
-
-# Install prod requirements
-RUN pip install --no-cache-dir -r requirements-demo.txt
-
-EXPOSE 8000
-
-ENTRYPOINT ["./run.sh"]
-
-
```

Comment:
Since we no longer have the `demo` stage, I think we should remove the `common` stage as well.

Reply:
Fair enough.